cfgov/jinja2/v1/ask-cfpb/landing-page.html

```diff
@@ -37,48 +37,47 @@ <h2 class="u-show-on-mobile u-mt30">
-                                        </a>
-                                    </h2>
-
-                                    <ul class="m-list question_list m-list__flush-left u-mt10 u-mb30">
-                                        {% for page in portal.featured_answers %}
-                                            <li class="m-list_item">
-                                            {{ partially_styled_link.render({
-                                                'plain_text': page.question,
-                                                'underlined_text': _('Read answer'),
-                                                'url': page.url })
-                                            }}
-                                            </li>
-                                        {% endfor %}
-                                    </ul>
-
-                                    <div class="content-l_footer__equal-height">
-                                        <p class="short-desc u-mb0">
-                                            <a href="{{ portal.url }}">
-                                               {{ _('Learn more about') }} {{ portal.title|lower }}
-                                            </a>
-                                        </p>
-                                    </div>
-
-                                </article>
+            <div class="u-hide-on-mobile question-categories">
+
+                {#
+                    TODO: This can use the standard card-group.html template
+                            if the partially styled links are removed.
+                #}
+                <div class="o-card-group o-card-group__column-2">
+                    <div class="o-card-group_cards">
+
+                    {% for portal in portal_cards %}
+                        <article class="m-card">
+                            <h2 class="m-card_heading">
+                                <a href="{{ portal.url }}">
+                                    <div class="m-card_icon">{{ category_icon.render( portal.topic ) }}</div>
+                                    <span>{{ portal.title }}</span>
+                                </a>
+                            </h2>
+
+                            <ul class="m-list question_list m-list__flush-left u-mt10 u-mb30">
+                                {% for page in portal.featured_answers %}
+                                    <li class="m-list_item">
+                                    {{ partially_styled_link.render({
+                                        'plain_text': page.question,
+                                        'underlined_text': _('Read answer'),
+                                        'url': page.url })
+                                    }}
+                                    </li>
+                                {% endfor %}
+                            </ul>
+
+                            <div class="content-l_footer__equal-height">
```

Comment:
Should this have the class `m-card_footer` instead?

Reply:
Yes! Fixed in https://github.com/cfpb/cfgov-refresh/pull/5538/commits/4cc683577077f1295120ea49eabda10a04ab785b